templates/website/about-qa.html

```diff
@@ -345,7 +345,7 @@
 <li>You made a typo when you input your email address, so our confirmation email has gone to the wrong place.</li>
 <li>You used a different email address from the one you are checking for our mail. You'd be surprised how often this happens!</li></ul>
 
-<p>If you cannot find our message in your spam folder, and you have looked in any alternative email accounts that you might have used, please <a href="/about-contact">let us know</a> - we can track down your original message and send it off from our end.</p>
+<p>If you cannot find our message in your spam folder, and you have looked in any alternative email accounts that you might have used, please <a href="/about-contact">let us know</a> - we can track down your original message and send it off from our end. If at any point in this process you decide that you don't want to receive any more emails from us, please email <a href="mailto:support@writetothem.com">support@writetothem.com</a>.</p>
```

Comment:
Oops, I just added it to the wrong section - I meant to stick it at the end of https://www.writetothem.com/about-qa#stop